wm-system/src/main/kotlin/com/ajaxproject/warehouse/service/WaybillServiceImpl.kt

```diff
@@ -123,6 +132,22 @@ class WaybillServiceImpl(
                     }
             }
 
+    private fun WaybillDataDto.mapToCreatedEventProto(): WaybillCreatedEvent =
+        WaybillCreatedEvent.newBuilder().apply {
+            waybill = Waybill.newBuilder().apply {
+                id = this@mapToCreatedEventProto.id
+                customerId = this@mapToCreatedEventProto.customer.id
+                date = Timestamp.newBuilder()
+                    .setSeconds(
+                        this@mapToCreatedEventProto.date
+                            .atStartOfDay()
+                            .atOffset(ZoneOffset.UTC)
+                            .toEpochSecond()
+                    )
+                    .build()
+            }.build()
+        }.build()
```

Comment:
```suggestion
        WaybillCreatedEvent.newBuilder().also { event ->
            event.waybillBuilder.also { waybill ->
                waybill.id = id
                waybill.customerId = customer.id
                waybill.dateBuilder.seconds = date.atStartOfDay().atOffset(ZoneOffset.UTC).toEpochSecond()
            }
        }.build()
```

Looks better, dosn't it?

Reply:
Done